core/src/main/java/com/linecorp/armeria/common/http/HttpResponseAggregator.java

```diff
@@ -38,7 +38,7 @@
     public void onNext(HttpObject o) {
         if (o instanceof HttpHeaders) {
             final HttpHeaders headers = (HttpHeaders) o;
-            if (headers.status().codeClass() == HttpStatusClass.INFORMATIONAL) {
+            if (headers.status() != null && headers.status().codeClass() == HttpStatusClass.INFORMATIONAL) {
```

Comment:
Extracted local variable and also added Nullable annotation which I had meant to do.

But to clarify - since the result of status() is memoized, I guess either way would be fine? If it looks like a getter and quacks like a getter, I'd assume it's a getter ;)